boa/src/vm/code_block.rs

```diff
@@ -35,12 +39,15 @@ unsafe impl Readable for f32 {}
 unsafe impl Readable for f64 {}
 
 /// The internal representation of a JavaScript function.
+///
+/// A CodeBlock is generated for each function compiled by the [ByteCompiler](crate::bytecompiler::ByteCompiler).
+/// It stores the bytecode and the other attributes of the function.
 #[derive(Debug, Trace, Finalize)]
 pub struct CodeBlock {
     /// Name of this function
     pub(crate) name: JsString,
 
-    /// The length of this function.
+    /// The number of parameters expected.
```

Comment:
`parameters` should be `arguments`, the function expects to be called with N arguments because it was declared with N parameters.